bind.go

```diff
@@ -98,6 +98,26 @@ func rebindBuff(bindType int, query string) string {
 	return rqb.String()
 }
 
+func asSliceToExpand(i interface{}) (v reflect.Value, ok bool) {
+	if i == nil {
+		ok = false
```

Comment:
This function is shorter and much easier to understand if it does not use naked returns.

```
func asSliceToExpand(i interface{}) (v reflect.Value, ok bool) {
	if i == nil {
		return nil, false
	}

	v = reflect.ValueOf(i)
	t := reflectx.Deref(v.Type())

	// []byte is a driver.Value type so it should not be expanded
	if t.Kind() != reflect.Slice || t == reflect.TypeOf([]byte{}) {
		return nil, false
	}
	return v, true
}
```

Also, a name like `asSliceForIn` might be clearer; "expand" is a bit vague.

Reply:
Thanks for the naming idea -- `asSliceForIn` is much better. I updated the named returns' style as well, but overall made it a little longer by splitting the slice and `[]byte` checks into multiple lines; coming back to it after 3 months, it just felt too dense to understand what was going on.